analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -544,7 +544,7 @@ def check_config_file(args):
 
     if 'config_file' in args and not os.path.exists(args.config_file):
         LOG.error("Configuration file '%s' does not exists.",
-                  args.config)
+                  args.config_file)
```

Comment:
Please fix the expression `if 'config_file' not in args:`
 to this  `if not 'config_file' in args or not os.path.exists(args.config_file):`
`config_file` will be always in args so the function will not early return here.